.github/workflows/pylama.yml

```diff
@@ -0,0 +1,28 @@
+name: Run linting
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
With pylama there is no use for multiple python version. Project uses version 3.11, we don't support others so please remove other versions